### PR TITLE
Add CNF WG member roles to charter

### PR DESCRIPTION
**Pull request by taylor**
Add CNF WG member roles to governance section of charter
- Chairs
- Tech Leads
- Everyone else

### CODE REVIEW

**Comment by iawells** [comment]
LGTM

**Review by jeffsaelens — APPROVED** [review]
Clear and concise.

**Review by xmulligan — APPROVED** [review]

**Review by tomkivlin — APPROVED** [review]

**Review by Smitholi67 — APPROVED** [review]

**Review by rabi-abdel — APPROVED** [review]

**Comment by rabi-abdel** [comment]
is the plan to also include details for the election process? that certainly can be on a separate PR

**Comment by xmulligan** [comment]
@rabi-abdel yes, that will be a separate PR

File: charter.md [hunk 1]
```diff
@@ -72,7 +72,51 @@ Deliverables
...
+-->
+
+
+---
+
+
+## Interested Parties
 
 - Bell Canada
 - Charter Communications
```
Comment (iawells) [inline charter.md:122]:
```suggestion
- Charter Communications
- Cisco
```

**Review by iawells — CHANGES_REQUESTED** [review]

File: charter.md (hunk 1 above)
Comment (electrocucaracha) [reply charter.md:122]: Technically speaking, this suggestion should have its own PR

Comment (iawells) [reply charter.md:122]: I mean, I can put it in as another pull request but I'm not sure I see why it's worth the extra effort (and I can't put it up until this is committed, I think?).

As far as I can see there's nothing special about the participants list here that stops us filling it out in the initial commit - it's not like they're specifically represented by the committer or anything - so we could check on the meeting to see if anyone else wants adding in this initial phase and save ourselves the extra step.

**Review by CsatariGergely — APPROVED** [review]

**Pull request merged**